@types/index.d.ts

```diff
@@ -33,3 +29,56 @@ type Attribute = {
   trait_type: string;
   value: string;
 };
+
+type Release = {
+  id: string;
```

Comment:
Should you infer cannot be null with a ! on the id?

Reply:
Nice spot. You only need to use the ! when inferring it can't be null in GraphQL. In typescript it's not null by default.